charms/katib-db-manager/src/charm.py

```diff
@@ -165,123 +168,162 @@ def _on_mysql_relation(self, event):
         try:
             relation = self.model.get_relation("relational-db")
             if relation:
-                self.logger.warnning(
-                    "Failed to create mysql relation due to existing relational-db relation."
+                self.logger.warning(
+                    "Up-to-date database relation relational-db is already established."
+                )
+                self.logger.warning(f"{MYSQL_WARNING} {UNBLOCK_MESSAGE}")
+                self.model.unit.status = BlockedStatus(
+                    f"{MYSQL_WARNING} {UNBLOCK_MESSAGE} See logs"
```

Comment:
1. I think message is too long for the screen.
2. maybe we should emphasize remove only `mysql` ?

Reply:
Looks like it fits. We also duplicate it in logs.

Reply:
I got it shorter:
```
MYSQL_WARNING = "Relation mysql is deprecated."
UNBLOCK_MESSAGE = "Remove deprecated mysql relation to unblock."
. . .
self.logger.error(f"{MYSQL_WARNING} {UNBLOCK_MESSAGE}")
self.model.unit.status = BlockedStatus(f"{UNBLOCK_MESSAGE} See logs")
```